sql/src/ast.rs

```diff
@@ -26,6 +26,32 @@ pub enum Statement {
     Exists(ExistsTable),
 }
 
+#[derive(Debug, PartialEq)]
+pub struct TableName(ObjectName);
+
+impl TableName {
+    pub fn is_empty(&self) -> bool {
+        self.0 .0.is_empty()
+    }
+}
+
+impl ToString for TableName {
+    fn to_string(&self) -> String {
+        self.0
+             .0
+            .iter()
+            .map(|ident| ident.value.as_str())
+            .collect::<Vec<_>>()
+            .join(".")
```

Comment:
```suggestion
    fn to_string(&self) -> String {
        self.0
            .0
            .iter()
            .map(|ident| ident.value.as_str())
            .collect::<Vec<_>>()
            .join(".")
```

Reply:
same with before.

Reply:
I can understand the first one but this is pretty strange to me. anyway it goes out of the scope of this PR.